yjit/src/codegen.rs

```diff
@@ -5416,6 +5417,33 @@ fn gen_send_cfunc(
         }
     }
 
+    // Log the name of the method we're calling to,
+    // note that we intentionally don't do this for inlined cfuncs
+    #[cfg(feature = "stats")]
```

Comment:
Is it intentional that we enable this on stats (or dev) build instead of `--yjit-stats`? Did you mean `if get_option!(gen_stats) {`?